spec.md

```diff
@@ -968,8 +968,8 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| dashboard_url | string | The URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. If present, MUST be a non-empty string. |
-| operation | string | For asynchronous responses, Service Brokers MAY return an identifier representing the operation. The value of this field MUST be provided by the Platform with requests to the [Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances) endpoint in a percent-encoded query parameter. If present, MUST be a non-empty string. |
+| dashboard_url | string | The URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. If present, MUST be a string or null. |
+| operation | string | For asynchronous responses, Service Brokers MAY return an identifier representing the operation. The value of this field MUST be provided by the Platform with requests to the [Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances) endpoint in a percent-encoded query parameter. If present, MAY be null. |
```

Comment:
When operation is null, does CF send any operation at all on the last_op call?

Reply:
I don't believe it does.

(I think [this](https://github.com/cloudfoundry/cloud_controller_ng/blob/master/lib/services/service_brokers/v2/client.rb#L243) is the relevant piece of code;  if the broker doesn't provide an operation (i.e. `parsed_response['operation']` is `nil`) then CC will return `nil` for `broker_provided_operation`.)

Reply:
For bindings there's a 10k limit on the size of this field - but that limit isn't mentioned here. Is CF inconsistent on this or does the text here need to include that size limit as well?

Reply:
@duglin sorry for the delay - you're quite right, CC enforces a 10k char limit. I'll add it to the PR.